src/applications/vaos/services/vaos/index.js

```diff
@@ -146,9 +153,33 @@ export function getSchedulingConfigurations(locationIds, ccEnabled = null) {
   ).then(parseApiList);
 }
 
-export function getAvailableV2Slots(facilityId, clinicId, startDate, endDate) {
+export function getAvailableV2Slots(
+  facilityId,
+  clinicId,
+  startDate,
+  endDate,
+  convertToUtc = false,
+) {
+  const start = convertToUtc
+    ? startDate.toISOString()
+    : format(
+        isDate(startDate)
+          ? startOfMonth(startDate)
+          : startOfMonth(parseISO(startDate)),
+        "yyyy-MM-dd'T'HH:mm:ssXXX",
+      );
+  const end = convertToUtc
+    ? endDate.toISOString()
+    : format(
+        isDate(endDate)
+          ? lastDayOfMonth(endDate)
+          : lastDayOfMonth(parseISO(endDate)),
+        "yyyy-MM-dd'T'HH:mm:ssXXX",
+      );
   return apiRequestWithUrl(
-    `/vaos/v2/locations/${facilityId}/clinics/${clinicId}/slots?start=${startDate}&end=${endDate}`,
+    `/vaos/v2/locations/${facilityId}/clinics/${clinicId}/slots?start=${encodeURIComponent(
```

Comment:
The 'encodeURIComponent' call actually fixes the error encountered when the offset is positive '+'. This approach should be used for the appointment calls as well.